build-test-deploy.yml

```diff
@@ -0,0 +1,166 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- main
+
+variables:
+  - group: s3PushInstallers
+  - group: pipeline-tokens
+  - name: configuration
+    value: Release
+
+pool: 'vs17_2'
```

Comment:
Do we need to use a self hosted pool, or will `windows-2022` do?

Reply:
Are you referring to making this a github action or do we have pipelines with `windows-2022` available on azure?